scripts/performance-tests-kontrol.sh

```diff
@@ -62,15 +70,15 @@ master_shell() {
 
 feature_shell "poetry install && poetry run kevm-dist --verbose build plugin haskell foundry --jobs 4"
```

Comment:
Should the build targets get prefixed here, as in the `kevm` script?
```suggestion
feature_shell "poetry install && poetry run kevm-dist --verbose build evm-semantics.plugin evm-semantics.haskell foundry --jobs 4"
```

Reply:
ah probably. im just testing them before merging so will correct